Lab1_Operationalizing_Pytorch_with_Mlflow/Makefile

```diff
@@ -0,0 +1,41 @@
+clean: ## Clean
+	rm -rf logs
+	rm -rf model_store
+	rm -rf artifacts
+	rm -rf labml_logs
+	rm mlflow.db
+
+train:
+	PYTHONPATH=$$PYTHONPATH:./src python -m bert_classifier.train
+
+model_store: ## Create empty model store
+	mkdir model_store
+
+torchserve-start: model_store ## Start torch serve
+	torchserve --start --model-store model_store
+
+torchserve-stop: ## Stop torch serve
+	torchserve --stop
+
+deploy:  ## Deploy trained model
+	PYTHONPATH=$$PYTHONPATH:./src python -m bert_classifier.deploy
+
+mlflow-ui:	## Start MLFlow UI
+	mlflow ui --backend-store-uri sqlite:///mlflow.db \
+		--default-artifact-root ./artifacts
+
+mlflow-server: ## Start MLFlow server
+	mlflow server \
+		--backend-store-uri sqlite:///mlflow.db \
+		--default-artifact-root ./artifacts \
+		--host localhost \
```

Comment:
This is great for local dev, would it be useful to have one for the distributed version?